README.md

```diff
@@ -48,12 +48,13 @@ Then save your lambda and test it with a test event!
 | Node.js version | ARN |
 | --- | --- |
 | 10.15.3 | `arn:aws:lambda:<region>:553035198032:layer:nodejs10:12` |
-| 12.0.0  | `arn:aws:lambda:<region>:553035198032:layer:nodejs12:1` |
+| 12.1.0  | `arn:aws:lambda:<region>:553035198032:layer:nodejs12:2` |
```

Comment:
@mhart FYI I just guessed at the ARN's final number here, assumed it was sequential.